docs/api-reference/full-text/boosting.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Boosting
+---
+
+## Basic Usage
```

Comment:
Ah, I just saw this. I think we should reference this page in the "Boost" query. I expect many people who find the "Boost" query to be looking for this functionality.